src/lib/components/support.svelte

```diff
@@ -5,17 +5,29 @@
     import { isSupportOnline, showSupportModal } from '$routes/(console)/wizard/support/store';
     import { trackEvent } from '$lib/actions/analytics';
     import { localeShortTimezoneName, utcHourToLocaleHour } from '$lib/helpers/date';
-    import { upgradeURL } from '$lib/stores/billing';
+    import { plansInfo } from '$lib/stores/billing';
     import { Card } from '$lib/components/index';
     import { app } from '$lib/stores/app';
-    import { currentPlan } from '$lib/stores/organization';
+    import { currentPlan, type Organization, organizationList } from '$lib/stores/organization';
     import { isCloud } from '$lib/system';
+    import { base } from '$app/paths';
 
     export let show = false;
 
     export let showHeader = true;
 
-    $: hasPremiumSupport = $currentPlan?.premiumSupport ?? false;
+    $: hasPremiumSupport = $currentPlan?.premiumSupport ?? allOrgsHavePremiumSupport ?? false;
+
+    $: allOrgsHavePremiumSupport = $organizationList.teams.every(
+        (team) => $plansInfo.get((team as Organization).billingPlan)?.premiumSupport
+    );
+
+    // there can only be one free organization
+    $: freeOrganization = $organizationList.teams.find(
+        (team) => !$plansInfo.get((team as Organization).billingPlan)?.premiumSupport
+    );
+
+    $: upgradeURL = `${base}/organization-${freeOrganization.$id}/change-plan`;
```

Comment:
do we need the one in the billing store if we have this here? Maybe it would be best to consolidate?

Reply:
Yep, we still need the other one because it uses the `organization` available from either page or the derived store. On `account` route, there's no `organization` access available. We could make this a function that has a default value but that goes outside the scope of this one and would require a refactor in some other places too.